models/src/main/java/com/vimeo/networking2/Video.kt

```diff
@@ -185,10 +189,11 @@ data class Video(
     /**
      * The status code for the availability of the video. This field is deprecated in favor
      * of [upload] and [transcode].
+     * @see Video.statusType
      */
     @Json(name = "status")
-    @Deprecated("This field is deprecated in favor of upload and transcode.")
-    val status: VideoStatusType? = null,
+    @Deprecated("This property is deprecated in favor of upload and transcode.")
```

Comment:
My view is that if a property is still available via the API, we should support it, even if it is deprecated. An additional reason to keep it is that we still heavily use the property in our Android application. 

Maybe @msya can weigh in on what he thinks.

Reply:
For v1 of models, the goal is to have parity with our current models. I prefer keeping the status field for now as our app uses it. The API spec will in the future specify a deprecated property for fields and it will specify the version in which it was deprecated. This will help in annotating which fields should be deprecated. There is a ticket in the queue to handle API versioning and this problem would be addressed as part of that ticket.